apm-agent-plugins/apm-grpc/apm-grpc-plugin/src/main/java/co/elastic/apm/agent/grpc/ChannelInstrumentation.java

```diff
@@ -75,15 +74,15 @@
     public static Object onEnter(@Advice.This Channel channel,
                                  @Advice.Argument(0) MethodDescriptor<?, ?> method) {
 
-        return helper.startSpan(tracer.getActive(), method, channel.authority());
+        return GrpcHelper.getInstance().startSpan(tracer.getActive(), method, channel.authority());
```

Comment:
An alternative that also works nicely is to create a separate static inner class for the advice and to store the helper as in a private static final variable. That has two advantages: No risk of eagerly loaded method parameters (such as ClientCall) as the advice class will not be loaded when the method matchers execute, and avoiding a static method call to get the helper instance.

That's definitely not a blocker for this PR.